### PR TITLE
origin addEventListener/removeEventListener should be called without eventListener

### PR DESCRIPTION
**Pull request by Brooooooklyn**
Should I commit amend or open a new pr?

### CODE REVIEW

**Comment by Brooooooklyn** [comment]
I tried to fix https://github.com/angular/zone.js/issues/198 here, please check @vicb

File: test/patch/element.spec.js [hunk 1]
```diff
@@ -114,17 +114,25 @@ describe('element', function () {
   });
 
   it('should have no effect while calling addEventListener without listener', function () {
+    var originAddEventListenerFn = zone.addEventListener;
```
Comment (vicb) [inline test/patch/element.spec.js]: A nicer way to do that would be to fork the zone installing the spy at that time.

While your are modifying the test, please also check the args (`toHaveBeenCalledWith`)

**Comment by vicb** [comment]
@Brooooooklyn thanks for the fix, it looks good to me - please update the tests according to the comments. Thanks.

**Comment by Brooooooklyn** [comment]
updated, please check

**Comment by vicb** [comment]
landed as 26e7f5177a06b28e9c4478eab5c6b95cd79da37f

Thanks @Brooooooklyn (you might want to add `fixes #<gh issue id>` when you fix gh issues).

**Pull request closed without merging**